### DISCUSSION TITLE
jasmine expects fails after wdio packages update to last version

### DISCUSSION THREAD

**Discussion by NavyMint** [webdriverio/webdriverio#7465] [Q&A] [answered]
Hi, guys
I've just updated wdio packages to latest version and all specs starts to fail on expect statements with next errors:

TypeError: expect(...).toEqual is not a function
TypeError: expect(...).toBe is not a function

test case:
```
it('test', () => {
        const test = 123;
        expect(test).toBe(123);
    });
```

Can you please suggest us what can be the problem here?

**Comment by christian-bromann** [comment, marked as answer]
This is a bug, please stay on the old version just for now.

**Reply by christian-bromann** [reply]
see https://github.com/webdriverio/webdriverio/issues/7466

**Reply by NavyMint** [reply]
got it, thank you, following issue
